oda_api/api.py

```diff
@@ -384,6 +381,17 @@ def generate_session_id(self, size=16):
         chars = string.ascii_uppercase + string.digits
         return ''.join(random.choice(chars) for _ in range(size))
 
+    @classmethod
```

Comment:
so we already a hash in `unique_response_json_fn` merged, they need to bereconciled somehow.

Reply:
I guess `unique_response_json_fn` can simply use the function `make_hash`